server/src/main/java/fr/uga/l3miage/tp2/exo2/models/Employe.java

```diff
@@ -0,0 +1,19 @@
+package fr.uga.l3miage.tp2.exo2.models;
+
+import javax.persistence.*;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name = "employe")
+public class Employe {
```

Comment:
```suggestion
public abstract class Employe {
```

la classe mère doit être abstraite car sinon tu permet l'intanciation quelconque d'un employer sans savoir son type